Phylum/Phylum.json

```diff
@@ -0,0 +1,150 @@
+                    "title": "# Threats by Ecosystem",
+                    "visualSettings": "{\"general\":{\"mode\":\"distribution\",\"type\":\"pie\",\"displayType\":\"default\",\"roundDataPoints\":true,\"fillOpacity\":1,\"startAngle\":270,\"innerRadius\":\"30%\",\"maxNumOfSlices\":10},\"title\":{\"fontSize\":14},\"legend\":{\"enabled\":true,\"verticalAlign\":\"bottom\",\"fontSize\":12,\"maxHeight\":50,\"showAsTable\":false,\"wrap\":true},\"axes\":{\"axisX\":{\"titleFontSize\":12,\"labelFontSize\":12},\"axisY\":{\"titleFontSize\":12,\"labelFontSize\":12}},\"series\":{}}",
+                    "keepVisualSettingsConsistentWithParent": true,
+                    "panelType": "SumoSearchPanel",
+                    "queries": [
+                        {
+                            "transient": false,
+                            "queryString": "cat path://\"/Library/Users/pete@phylum.com/phylum-threat-data/threat-test\" | count by ecosystem",
+                            "queryType": "Logs",
+                            "queryKey": "A",
+                            "metricsQueryMode": null,
+                            "metricsQueryData": null,
+                            "tracesQueryData": null,
+                            "spansQueryData": null,
+                            "parseMode": "Auto",
+                            "timeSource": "Message",
+                            "outputCardinalityLimit": 1000
+                        }
+                    ],
+                    "description": "Show a count of software supply chain threats by ecosystem",
+                    "timeRange": null,
+                    "coloringRules": null,
+                    "linkedDashboards": []
+                },
+                {
+                    "id": null,
+                    "key": "panelPANE-509350EAA787E84B",
+                    "title": "Malicious Packages Detected In Your Environment by Ecosystem",
+                    "visualSettings": "{\"general\":{\"mode\":\"distribution\",\"type\":\"pie\",\"displayType\":\"default\",\"roundDataPoints\":true,\"fillOpacity\":1,\"startAngle\":270,\"innerRadius\":\"30%\",\"maxNumOfSlices\":10},\"title\":{\"fontSize\":14},\"legend\":{\"enabled\":true,\"verticalAlign\":\"bottom\",\"fontSize\":12,\"maxHeight\":50,\"showAsTable\":false,\"wrap\":true},\"axes\":{\"axisX\":{\"titleFontSize\":12,\"labelFontSize\":12},\"axisY\":{\"titleFontSize\":12,\"labelFontSize\":12}},\"series\":{}}",
+                    "keepVisualSettingsConsistentWithParent": true,
+                    "panelType": "SumoSearchPanel",
+                    "queries": [
+                        {
+                            "transient": false,
+                            "queryString": "$$logsrc  |\nparse \"hash: *\" as candidate_hash\n| lookup * from path://\"/Library/Users/pete@phylum.com/phylum-threat-data/threat-test\" on id=candidate_hash | count by ecosystem | sort by +_count | limit 2",
```

Comment:
Customer's lookup path would be different, so how will customer update this json? May be they have to give email as input or use variable.

Reply:
Hey @peterjmorgan - If you notice in your dashboard, the path you're using for your lookup table is tied to your account. This will need to be changed in every chart by customers as their paths will be different.

@himanshu219 - What do you think is the best approach here? Should Pete just document how to change it, or should we expose a template variable that they can just copy/paste their path in to? Or something else?

Reply:
@peterjmorgan could you please provide to me some data so that I can populate the lookup table (in my org) so as to test the dashboards.